### ISSUE TITLE
Issues running the sample application

### ISSUE THREAD

**Issue by marksgraham** [AnswerConsulting/flip-sample-application#2]
Hi,

I've encountered two issues when trying to get the sample app up and running.

1. I don't see this documented but pandas and boto3 need to be installed - an error is thrown when I run `start_app all` if these aren't installed.
2. After solving this I get a json-related error when I run `start_app all`. It looks like it is related to the model ID, which is down as         `"model_id": "{model_id}"` in `config_fed_server.json`. The message gets printed badly to terminal so it's quite hard to read but here it is:

```bash
 File "/home/mark/Envs/flip-sample-app/lib/python3.8/site-packages/nvflare/private/json_configer.py", line 99, in configure
                                                                                                                                                                              raise ConfigError("Config error in {}: {}".format(self.config_file_name, ex))
                   nvflare.fuel.common.excepts.ConfigError: Config error in /home/mark/projects/federated_learning/flip-sample-application/poc/server/startup/../run_1/app_server/config/config_fed_server.json: Error processing /home/mark/projects/federated_learning/flip-sample-application/poc/server/startup/../run_1/app_server/config/config_fed_server.json in JSON element components.#6: The model ID: None is not a valid UUID
                                                                                                                                                                                                   2022-07-25 19:59:00,017 - root - WARNING - FL server execution exception: Config error in /home/mark/projects/federated_learning/flip-sample-application/poc/server/startup/../run_1/app_server/config/config_fed_server.json: Error processing /home/mark/projects/federated_learning/flip-sample-application/poc/server/startup/../run_1/app_server/config/config_fed_server.json in JSON element components.#6: The model ID: None is not a valid UUID
                                                                                                                                                                            2022-07-25 19:59:00,018 - FederatedServer - INFO - Server app stopped.
```

Thanks,
Mark

**Comment by yorkeanswer** [comment]
Hi Mark,

These configuration values are templated in during runtime on FLIP - for this example we will need to set some default values.

@mehranrashid is currently putting together a PR which will resolve this. Once it is created I will ask him to update this ticket so you can see when it is merged.

Cheers,
Yorke

**Comment by yorkeanswer** [comment]
This issue will resolved in #4 👍
